_overviews/tutorials/binary-compatibility-for-library-authors.md

```diff
@@ -167,12 +167,60 @@ For example, the use of these language features are a common source of binary co
 in library releases:
 
 * Default parameter values for methods or classes
-* Case classes (work around the issue with [this pattern]({{ site.baseurl }}/scala3/book/domain-modeling-tools.html#changing-case-class-definition-in-a-compatible-manner))
+* Case classes
 
 You can find detailed explanations, runnable examples and tips to maintain binary compatibility in [Binary Compatibility Code Examples & Explanation](https://github.com/jatcwang/binary-compatibility-guide).
 
 Again, we recommend using MiMa to double-check that you have not broken binary compatibility after making changes.
 
+### Changing case class definition in a compatible manner
+
+Sometimes it is desirable to be able to change the definition of a case class (adding and/or removing fields) while still staying compatible with the users of the case class, i.e. not breaking the so called _binary compatibility_.
+
+To achieve that, follow this pattern:
+ * make the constructor `private`
+ * define `private` `unapply` function in the companion object (note that by doing that the case class looses the ability to be used in a pattern match)
```

Comment:
```suggestion
 * define a private `unapply` function in the companion object (note that by doing that the case class loses the ability to be used in a pattern match)
```

typo (loses) + "private" can be considered part of "private ... function" so I wouldn't use backticks here (it could also give a false impression that `private unapply` is valid syntax)

Reply:
(I'd untick the line above as well)